app/assets/stylesheets/govuk-component/govspeak/_magna-charta.scss

```diff
@@ -0,0 +1,306 @@
+        box-sizing: border-box;
+        border: 0 solid;
+      }
+
+      // Bars
+      .mc-bar-cell {
+        display: block;
+        text-align: left;
+        padding: $bar-padding 0;
+        margin-bottom: 1px;
+        color: $white;
+        text-indent: $bar-padding;
+      }
+
+      // Bar colours
+      @for $i from 0 to length($bar-colours) {
+        .mc-bar-#{$i + 1} {
+          background-color: nth($bar-colours, $i + 1);
+        }
+      }
+
+      // Negative bars
+      .mc-bar-negative {
+        text-align: right;
+        padding-right: $bar-padding;
+      }
+
+      .mc-value-overflow {
+        text-indent: -99999px !important;
+      }
+
+      // Bar labels
+      .mc-key-cell {
+        width: $label-width;
+        text-indent: 26px;
+        text-align: $label-align;
+        display: table-cell;
+        vertical-align: middle;
+      }
+    }
+  }
+
+  // STACKED CHARTS
+  &.mc-stacked {
+    .mc-stacked-header {
+      display: none; // Hide the header for the totals column
+   
+    }
+
+    .mc-th:nth-last-child(2) {
+      margin-bottom: 0;
+    }
+
+    .mc-stacked-total {
+      padding: $bar-padding 0;
+      background-color: transparent !important;
+      color: $black;
+      display: inline-block;
+      text-indent: 5px;
+    }
+
+    .mc-tbody .mc-bar-cell {
+      display: inline-block;
+    }
+  }
+
+  // COMPACT CHARTS
+  &.compact {
+    .mc-td.mc-bar-cell {
+      font-size: $compact-font-size;
+      text-indent: 2px;
+      padding: 0;
+    }
+  }
+
+  // Charts with multiple columns get different coloured bars
+  $len: length($bar-colours);
+
+  @for $i from 0 to $len {
+    .mc-tr .mc-td.mc-bar-cell.mc-bar-#{$i + 1} {
+      background-color: nth($bar-colours, $i + 1);
+    }
+
+    .mc-tr .mc-th.mc-key-#{$i + 1} {
+      border-left-color: nth($bar-colours, $i + 1);
+    }
+  }
+
+  .mc-td, .mc-th {
+    padding: 0;
+    padding-right: $bar-spacing;
+    margin-right: $stack-spacing;
+    border: 0 solid;
+
+    &.mc-bar-cell {
+      display: block;
+      background-color: #ebebeb; // Just for testing
+      text-align: left;
+      padding: $bar-padding 0;
+      margin-bottom: 1px;
+      color: $white;
+      text-indent: 4px;
+    }
+  }
+
+  .mc-bar-outdented {
+    span {
+      color: #111;
+    }
+  }
+
+  caption {
+    caption-side: $caption-side;
+  }
+
+  .mc-td.mc-key-cell {
+    width: $label-width;
+    text-indent: 26px;
+    text-align: $label-align;
+    display: table-cell;
+    vertical-align: middle;
+  }
+
+  // OUTDENTED BAR VALUES
+  &.mc-outdented {
+    .mc-tbody {
+      .mc-tr {
+        .mc-bar-cell {
+          color: #111;
+        }
+
+        .mc-bar-negative {
+          text-align: left;
+        }
+      }
+    }
+  }
+
+  &.mc-negative .mc-td.mc-key-cell {
+    padding-right: 25px;
+  }
+}
+
+.mc-outdented .mc-bar-cell.mc-bar-negative {
+  text-align: left;
+}
+
+// .mc-bar-cell.mc-bar-negative.mc-bar-indented {
```

Comment:
Can we tidy up commented out code if we’re losing direct blame relations anyway?